### PR TITLE
[QA] SISRP-23173 - Header popover shows CNP and Registration alert but no Status Block Card exists

### PR DESCRIPTION
**Pull request by davidlimcheng**
https://jira.berkeley.edu/browse/SISRP-23173

* Apologies for this, prior sloppy code on my part.  This is why coding shouldn't happen past 10pm =\

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/pages/academicsController.js [hunk 1]
```diff
@@ -199,7 +199,7 @@ angular.module('calcentral.controllers').controller('AcademicsController', funct
     // The university_requirements widget is also used on Advising Dashboard.
     $scope.academics.universityRequirements = $scope.requirements;
     $scope.showStatusAndBlocks = !$scope.filteredForDelegate &&
-                                 ($scope.hasRegStatus ||
+                                 ($scope.hasRegstatus ||
```
Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/pages/academicsController.js]: @davidlimcheng Despite our rush, I'd prefer `hasRegStatus` across the board. Lowercase `status` seems wrong.

Comment (davidlimcheng) [inline src/assets/javascripts/angular/controllers/pages/academicsController.js]: @johncrossman I like the cut of your jib.  I don't really know what that means, but agreed- amending!

**Comment by johncrossman** [comment]
With sailing references in mind, I like the new tack. Thanks, @davidlimcheng  Waiting on T.

**Comment by johncrossman** [comment]
My comment was addressed. Thanks, @davidlimcheng

**Pull request merged**